### PR TITLE
Use the term Baggage in place of Trace Attributes

### PR DESCRIPTION
**Pull request by bensigelman**
C.f. https://github.com/opentracing/opentracing.github.io/issues/67:

"Baggage" is the *set* of Baggage items; so

  "Trace Attribute" == "Baggage Item"

and

  "Trace Attributes" == "Baggage"

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -51,16 +51,16 @@ Every Span may also have zero or more key/value **Tags**, which do not have time
...
 
-**Trace Attributes** are key/value pairs stored in a Span and propagated _in-band_ to all future child Spans. Given a full-stack OpenTracing integration, Trace Attributes enable powerful functionality by transparently propagating arbitrary application data: for example, an end-user id may be added as a Trace Attribute in a mobile app, propagate (via the distributed tracing machinery) into the depths of a storage system, and recovered at the bottom of the stack to identify a particularly expensive SQL query.
+**Baggage** is a set of key/value pairs stored in a Span and propagated _in-band_ to all future child Spans: in this way, the "Baggage" travels with the trace, hence the name. Given a full-stack OpenTracing integration, Baggage enables powerful functionality by transparently propagating arbitrary application data: for example, an end-user id may be added as Baggage in a mobile app, propagate (via the distributed tracing machinery) into the depths of a storage system, and recovered at the bottom of the stack to identify a particularly expensive SQL query.
```
Comment (yurishkuro) [inline spec.md]: `s/future child Spans/future children Spans/` ?

Comment (yurishkuro) [inline spec.md]: > an end-user id may be added as Baggage in a mobile app

`a/Baggage/Baggage item/` ?

File: spec.md [hunk 2]
```diff
@@ -74,19 +74,19 @@ The `Span` interface must have the following capabilities:
...
 - Add a new log event to the `Span`, accepting an event name `string` and an optional structured payload argument. If specified, the payload argument may be of any type and arbitrary size, though implementations are not required to retain all payload arguments (or even all parts of all payload arguments). An optional timestamp can be used to specify a past timestamp. **(py: `log`, go: `Log`)**
-- Set a trace attribute, which is a simple string:string pair. Note that newly-set trace attributes are only guaranteed to propagate to *future* children of the given `Span`. See the diagram below. **(py: `set_trace_attribute`, go: `SetTraceAttribute`)**
-- Get a trace attribute by key. **(py: `get_trace_attribute`, go: `TraceAttribute`)**
+- Set a Baggage item, represented as a simple string:string pair. Note that newly-set Baggage items are only guaranteed to propagate to *future* children of the given `Span`. See the diagram below. **(py: `set_baggage_item`, go: `SetBaggageItem`)**
```
Comment (bensigelman) [inline spec.md]: (could also do `add_baggage()`, but `set_baggage()` seems wrong semantically)

Comment (yurishkuro) [inline spec.md]: `set_baggage_item` sgtm

**Comment by yurishkuro** [comment]
minor comments, lgtm

**Comment by bensigelman** [comment]
All done, thanks.

**Pull request merged**